server/src/main/java/io/crate/expression/scalar/NumTermsPerDocQuery.java

```diff
@@ -96,35 +97,15 @@ private static IntUnaryOperator getNumTermsPerDocFunction(LeafReader reader, Str
 
             case StringType.ID:
             case CharacterType.ID:
-                return numValuesPerDocForString(reader, fqColumn);
-
+            case BitStringType.ID:
             case IpType.ID:
-                return numValuesPerDocForIP(reader, fqColumn);
+                return numValuesPerDocForString(reader, fqColumn);
```

Comment:
behaving for `IP` same way as for STRING/BIT looks fine since their mappers use `SortedSetDocValuesField` - but on the other hand, numValuesPerDocForIP is old code, not sure why it stopped working and causing 
`NullPointerException[Cannot invoke "org.apache.lucene.index.SortedSetDocValues.advanceExact(int)" because "docValues" is null]`

Reply:
thx for the explanation!